iis/moduleconfig.h

```diff
@@ -69,7 +69,8 @@ class MODSECURITY_STORED_CONTEXT : public IHttpStoredContext
 
 	void*			  m_Config;
 
-    bool configLoadingFailed = false;
+	time_t configFailTime;
```

Comment:
Minor: let's default-initialize this to e.g. `0` so that we have some meaningful value.
While current execution path doesn't rely on this we should be future-proof.